packages/test/test-utils/src/testObjectProvider.ts

```diff
@@ -257,6 +259,11 @@ export class TestObjectProvider implements ITestObjectProvider {
         return this._loaderContainerTracker;
     }
 
+    /** Caution: This method is not designed to be user friendly, the this._logger is reset after each test */
```

Comment:
I don't understand this comment.  "is reset after each test" -- do you mean that the typical lifetime of the TestObjectProvider is just a single test?

Reply:
Yeah, in the `beforeEach` of each test, `provider.reset()` is called by default.